src/test/java/ru/training/at/hw8/TrelloBoardTests.java

```diff
@@ -0,0 +1,177 @@
+package ru.training.at.hw8;
+
+import static ru.training.at.hw8.constants.ResponseField.*;
+import static ru.training.at.hw8.core.TrelloServiceObject.*;
+import static ru.training.at.hw8.matchers.Matchers.*;
+
+
+import io.restassured.http.Method;
+import org.testng.annotations.AfterMethod;
+import org.testng.annotations.Test;
+import ru.training.at.hw8.beans.TrelloResponse;
+import ru.training.at.hw8.constants.PermissionLevel;
+import ru.training.at.hw8.core.DataProviderForTrello;
+
+public class TrelloBoardTests {
```

Comment:
GENERAL:
when there are too many boards, tests won't pass. You HAVE! to delete them after every test. If a test fails, the board is not deleted. Use @AfterMethod. There is a way to get the created board id and pass it as a parameter to the @AfterMethod.

Reply:
Okay, I just deleted it

Reply:
I didn't use @AfterMethod to delete boards? because I don't have ti delete them in @AfterMethod in the last test. because I have to do some extra actions after deleting the board in that test

Reply:
Or I just shouldn't do any asserts after delete action?

Reply:
I make tearDown() now